packages-ts/starknet-gauntlet-example/src/lib/contracts.ts

```diff
@@ -1,14 +1,25 @@
 import fs from 'fs'
-import { CompiledContract, json } from 'starknet'
+import { json } from 'starknet'
 
 export enum CONTRACT_LIST {
   EXAMPLE = 'example',
 }
 
-export const loadContract = (name: CONTRACT_LIST): CompiledContract => {
-  return json.parse(
-    fs.readFileSync(`${__dirname}/../../contract_artifacts/abi/${name}.json`).toString('ascii'),
-  )
+export const loadContract = (name: CONTRACT_LIST) => {
+  return {
+    contract: json.parse(
+      fs.readFileSync(
+        `${__dirname}/../../../../contracts/target/release/chainlink_${name}.sierra.json`,
+        'utf-8',
+      ),
+    ),
+    casm: json.parse(
+      fs.readFileSync(
+        `${__dirname}/../../../../contracts/target/release/chainlink_${name}.casm.json`,
+        'utf-8',
+      ),
+    ),
+  }
 }
```

Comment:
Since the paths are now flattened out, this `loadContract` function looks the same everywhere (except it's typed by `CONTRACT_LIST`). Let's extract it and share it across the files? Would make it easier to update